courses/lms/templates/static_templates/faq.html

```diff
@@ -47,23 +47,35 @@ <h1 style="text-align: left;">Frequently Asked Questions</h1>
+    <p><a href="https://microsoft.sharepoint.com/sites/Infopedia/search/pages/results.aspx?term=%22Microsoft%20Exchange%20Server%202016%22%20MOOC&scope=All">"Microsoft Exchange Server 2016"MOOC</a></p>
+    <p><a href="https://microsoft.sharepoint.com/sites/Infopedia/search/pages/results.aspx?term=%22Microsoft%20Project%22%20MOOC&scope=All">"Microsoft Project MOOC"</a></p>
+    </p><br>
+        <img src="/static/courses/images/infopedia_courses.png" alt="infopedia_courses_img" height="400">
+        <br>
+	<b>Q. Will my grades show in my Infopedia transcript?</b><br>
+	     <p>Yes.</p><br>    
     <b>Q. Will my manager see the courses I’ve taken in InfoPedia?</b><br>
         <p>Yes.</p><br>
     <b>Q. What is the minimum percentage required to pass the course?</b><br>
         <p>Each course can have a unique grading policy. Review the About page for the course to understand the requirement to pass the course.</p><br>
     <b>Q. How do I track my course progress?</b><br>
         <p>Open your course and select the Progress tab </p><br>
     <b>Q. How do I register?</b><br>
-        <p>To register, click launch on a course card on Infopedia. You will be registered and enrolled directly from Infopedia.</p><br>
+        <p>To register, click launch on a course card on Infopedia. You can register and enroll directly from Infopedia.</p><br>
     <b>Q. I’d like to try the Open edX experience, but no courses are available to me right now.</b>
         <p>The best way to get a sense of the user experience is by taking the edX Demo course. You can also find over 100 courses delivered by Microsoft that are open to the public on <a href="https://www.edx.org/school/microsoft">https://www.edx.org/school/microsoft.</a></p><br>
-    <b>Q. What do I do if I am having trouble viewing a video?</b><br>
-        <p>Please click on the download video button located below the video player. See screenshot.</p><br>
+    <b>Q.What do I do if I cannot access a course and I get an error saying my email address belongs to another account, and to try again with a different address?</b><br>
+	    <p> A. This is because you already created an account on the Azure Skills Initiative site using your @microsoft.com email address. You should log into the Azure site on <a href="https://openedx.microsoft.com"> https://openedx.microsoft.com</a> and change your email address under Account Settings. We recommend that you update the email address with a personal email address. After this, please try to launch your course again from MyLearning or the link you were provided.</p><br>
+	<b>Q. What do I do if I am having trouble viewing a video?</b><br>
+        <p>The video player will auto-select the best video quality option for your bandwidth. You can try a lower quality option, or you can download the video if that option is available. See screenshot.
+
+</p><br>
         <img src="/static/courses/images/download_video.png" alt="download_video_img" height="400">
```

Comment:
can you share the url of a fullstack deployment that has these changes? if not, swing by and we'll apply these changes to one of my fullstacks and we'll share that url to the team : )